### PR TITLE
Update FAF99301 Extra Life Plando.pnach

### PR DESCRIPTION
**Pull request by E-Ada93**
updates to Cern's updates circa 1:08 PM PST

### CODE REVIEW

File: FAF99301 Extra Life Plando.pnach [hunk 1]
```diff
@@ -299,9 +299,9 @@ patch=1,EE,11CE0222,extended,0000021A // Mansion Basement Corridor, Hi-Potion is
 
 //Popups
 patch=1,EE,11CE0636,extended,000001DF // Twilight Town Map, Twilight Town Map is now Ultimate Recipe
-patch=1,EE,11CE0606,extended,00000041 // Munny Pouch (Olette), Munny Pouch (Olette) is now Draw Ring
-patch=1,EE,11CE0612,extended,00000007 // Champion Belt, Champion Belt is now Mega-Elixir
-patch=1,EE,11CE061E,extended,00000174 // Medal, Medal is now Cursed Medallion
+patch=1,EE,11CE0606,extended,00000174 // Munny Pouch (Olette), Munny Pouch (Olette) is now Cursed Medallion
+patch=1,EE,11CE0612,extended,00000041 // Champion Belt, Champion Belt is now Draw Ring
+patch=1,EE,11CE061E,extended,00000041 // Medal, Medal is now Draw Ring
```
Comment (1234567890num) [inline FAF99301 Extra Life Plando.pnach]: This is to give extra reward if you manage to beat Yuffie (not too hard if you have Guard & Counterguard)

File: FAF99301 Extra Life Plando.pnach [hunk 2]
```diff
@@ -622,10 +622,8 @@ patch=1,EE,11C583E4,extended,00000134 // Dragoon
 
 //Boss Replacement
 // Marluxia Safety
-patch=1,EE,E004050C,extended,0032BAE0
-patch=1,EE,E0010166,extended,01CE0B6A
+patch=1,EE,E002050C,extended,0032BAE0
 patch=1,EE,11CE0B6A,extended,0000005C // Revert Dolan from Marluxia battle
-patch=1,EE,E0010166,extended,01CE0B6C
 patch=1,EE,11CE0B6C,extended,0000005D // Revert Goffy from Marluxia battle
```
Comment (1234567890num) [inline FAF99301 Extra Life Plando.pnach:627]: This is Marluxia's safety lines. Haven't been tested though.

File: FAF99301 Extra Life Plando.pnach [hunk 3]
```diff
@@ -763,13 +761,11 @@ patch=1,EE,11C57C88,extended,00000004 // Hook Bat
...
-patch=1,EE,E0060001,extended,0032BAE4
-patch=1,EE,E0050000,extended,0032BAE6
-patch=1,EE,E0040001,extended,0032BAE8
-patch=1,EE,E00104B8,extended,01C56BB0
+patch=1,EE,E005010D,extended,0032BAE0
+patch=1,EE,E0040001,extended,0032BAE4
+patch=1,EE,E0030000,extended,0032BAE6
+patch=1,EE,E0020001,extended,0032BAE8
 patch=1,EE,11C56BB0,extended,00000988
-patch=1,EE,E00104B8,extended,01C56B70
```
Comment (1234567890num) [inline FAF99301 Extra Life Plando.pnach]: Prevent crash when going into pier

File: FAF99301 Extra Life Plando.pnach [hunk 4]
```diff
@@ -1904,7 +1900,7 @@ patch=1,EE,21D11158,extended,000001D0 // Replacement Reward #2: Replacement Rewa
 //Enemy Replacement
 //Hades Escape
 patch=1,EE,E01B0506,extended,0032BAE0
-patch=1,EE,E01A006F,extended,0032BAE8
+patch=1,EE,E01A0003,extended,0032BAE8
```
Comment (1234567890num) [inline FAF99301 Extra Life Plando.pnach]: Make Yuffie actually invincible

File: FAF99301 Extra Life Plando.pnach [hunk 5]
```diff
@@ -2442,8 +2438,8 @@ patch=1,EE,01C6BDB4,extended,000001F4
 patch=1,EE,01C6BDB0,extended,000001F4
 //Dancer Marker Fix
 patch=1,EE,E0030305,extended,0032BAE0
-patch=1,EE,E0020016,extended,0032BAE8
-patch=1,EE,E0010003,extended,01C98F0C
+patch=1,EE,E0020045,extended,0032BAE8
+patch=1,EE,E0010000,extended,01C6BDB0
```
Comment (1234567890num) [inline FAF99301 Extra Life Plando.pnach]: Prevent softlock after beating Dancer

File: FAF99301 Extra Life Plando.pnach [hunk 6]
```diff
@@ -2459,8 +2455,8 @@ patch=1,EE,01C6BF3C,extended,00000007
 patch=1,EE,E0040F05,extended,0032BAE0
 patch=1,EE,E0030052,extended,0032BAE8
 patch=1,EE,11C5A0A8,extended,0000063C// Riku
-patch=1,EE,E0010302,extended,01C6C284// If Max HP Slot 4 = 770
-patch=1,EE,11C6C288,extended,00000000// HP Gate = 0HP
+patch=1,EE,E0010302,extended,01C6C01C// If Max HP Slot 4 = 302
+patch=1,EE,11C6C020,extended,00000000// HP Gate = 0HP
```
Comment (1234567890num) [inline FAF99301 Extra Life Plando.pnach]: Enable actually killing Riku

**Review by 1234567890num — CHANGES_REQUESTED** [review]
I really dunno how this works

File: FAF99301 Extra Life Plando.pnach (hunk 1 above)
Comment (E-Ada93) [reply FAF99301 Extra Life Plando.pnach]: oh, i was just erring on the side of caution and made both rewards the same..

File: FAF99301 Extra Life Plando.pnach (hunk 3 above)
Comment (E-Ada93) [reply FAF99301 Extra Life Plando.pnach]: so far pier didn't crash, but my current saves are in a post past pete state, i assumed this was updated via cern >_< edit: oh it crashed before past pete, that's my bad for being dumb

File: FAF99301 Extra Life Plando.pnach (hunk 2 above)
Comment (E-Ada93) [reply FAF99301 Extra Life Plando.pnach:627]: tested with both dying to Marluxia and then clearing the fight, no crashes or issues to note

**Pull request merged**